src/components/Sections/SectionWrapper.tsx

```diff
@@ -44,7 +44,7 @@ const SectionWrapper: React.FC<SectionProps> = ({
                 <Header as="h2" content={details.title} style={sectionStyles.title} />
               </Grid.Column>
               <Grid.Column floated="right" textAlign="right">
-                {extraSectionTitleContent && extraSectionTitleContent(section)}
```

Comment:
hmm also was wondering about this change, what was wrong with React.ReactNode vs React.FC<SectionState>

Reply:
I can revert that, I thought it was clearer this way. I did that because I felt more confident use React.FC, will check out what is the difference

Reply:
Reverted now: 65790b0